pkg/controller/v1beta1/inferenceservice/components/predictor.go

```diff
@@ -276,6 +409,39 @@ func (p *Predictor) Reconcile(isvc *v1beta1.InferenceService) (ctrl.Result, erro
 		return !utils.Includes(constants.ServiceAnnotationDisallowedList, key)
 	})
 
+	// Autoscaler should be ignored when multiNodeEnabled is true
+	workerObjectMeta = metav1.ObjectMeta{}
+	if multiNodeEnabled {
+		if annotations[constants.AutoscalerClass] != string(constants.AutoscalerClassExternal) {
+			isvc.Status.UpdateModelTransitionStatus(v1beta1.InvalidSpec, &v1beta1.FailureInfo{
+				Reason:  v1beta1.InvalidPredictorSpec,
+				Message: "when Multi Node is enabled, the autoscaler type must be 'external' only.",
+			})
```

Comment:
This logic should be moved to the validation webhook

Reply:
oops this is dead source.  I already added this to validation so I removed it.